msteams-platform/messaging-extensions/api-based-overview.md

```diff
@@ -11,9 +11,7 @@ ms.date: 10/19/2023
 # Build message extensions using API
 
 > [!NOTE]
->
-> * API-based message extensions only support search commands.
-> * API-based message extensions are available only in [public developer preview](../resources/dev-preview/developer-preview-intro.md).
+> API-based message extensions only support search commands.
 
 Message extensions built using API (API-based) use a web service to manage user requests and responses and don't require a bot registration or a Bot Framework. You can configure and deploy API-based message extensions using Developer Portal for Teams, Visual Studio Code, Teams Toolkit command line interface (CLI), or Visual Studio. API-based message extensions help your apps to interact directly with third-party data, apps, and services, enhancing its capabilities. With APIs for message extension, you can:
```

Comment:
* Remove "or a Bot Framework".
* For "or Visual Studio", I'm not sure whether developers can create a new API ME from VS directly. Please confirm this with Mao.
* For this statement "Perform actions on behalf of the user, for example, create a Contoso ticket.", I don't think this can be done with search based ME. Also check with Mao.